CentralComputing/Pod.h

```diff
@@ -12,7 +12,6 @@
 #pragma GCC diagnostic pop
 
 const int HEARTBEAT_GPIO = 37;
```

Comment:
Please move this to the `Utils::` namespace. We will likely add other GPIO pins later, and it makes more sense to have them in `Utils` than as a global variable